maven-embedder/src/main/java/org/apache/maven/cli/CLIManager.java

```diff
@@ -280,6 +280,12 @@ public CLIManager() {
                 .desc("Ineffective, only kept for backward compatibility")
                 .build());
 
+        // Adding this back to make Maven fail if used
```

Comment:
Please add info about conflict with `-l lr` - why we need it

Reply:
We need it not only to avoid conflicting `-l lr` but also to emit sane error message (so we really want to parse it from CLI but fail if present). So, I started a new "section" here (below the "ineffective section") for those that should fail.